.php-cs-fixer.dist.php

```diff
@@ -20,10 +20,10 @@
         'phpdoc_no_package' => false,
```

Comment:
If we want to make this update date from the current master should we also port 19ed8ace in?